packages/ui/src/bounty/queries/bounty.graphql

```diff
@@ -170,3 +170,14 @@ query GetBountyContributors(
     ...BountyContributionFields
   }
 }
+
+query GetLatestBountyByMemberId($memberId: ID) {
+  bounties(where: { creator: { id_eq: $memberId } }, orderBy: [createdAt_DESC], limit: 1) {
+    id
+    createdInEvent {
+      createdAt
+      inBlock
+      network
+    }
+  }
+}
```

Comment:
AFAIK there is no lock when creating a bounty. The bounty lock comes from anouncing an entry so the query should be:
```gql
bountyEntries(where: { stakingAccount_eq: $lockAccount }, orderBy: [createdAt_DESC], limit: 1) ...
```